frontend/src/UseFormSender.js

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+import axios from 'axios';
+
+const UseFormSender = (reqType, url) => {
+    const [error, setError] = useState(null);
+    const [loading, setLoading] = useState(false);
+    const [data, setData] = useState(null);
+
+    const cb = async (requestBody) => {
```

Comment:
use `req` and `res` everywhere instead of `request` and `response` please. It's very common like writing err for error :D